Includes/ftpConnection.cpp

```diff
@@ -38,6 +38,8 @@
 bool nxIsDriveMounted(char) { return true; }
 #endif
 
+#define FTP_CMD_BUFFER_SIZE (1024)
+#define FTP_DATA_BUFFER_SIZE (64 * 1024)
```

Comment:
Whats your reason for this exact sizes?

Reply:
I've added a comment. Hope it added some clarity. :)